src/grt/include/grt/GlobalRouter.h

```diff
@@ -152,7 +152,7 @@ class GlobalRouter
   int getMaxRoutingLayer() const { return max_routing_layer_; }
   void setMinLayerForClock(const int min_layer);
   void setMaxLayerForClock(const int max_layer);
-  void setCriticalNetsPercentage(float critical_nets_percentage);
+  void setCriticalNetsPercentage(int critical_nets_percentage);
```

Comment:
Should this not still be float so a user could set the percentage to e.g. 30.5%? It might be confusing when inputs of `30` and `30.1` provide the same result. I don't think `sta::check_percent` requires an int.